crates/sui-verifier/src/one_time_witness_verifier.rs

```diff
@@ -173,7 +173,9 @@ fn verify_init_single_param(
     let fn_sig = view.signature_at(fn_handle.parameters);
     if fn_sig.len() != 1 {
         return Err(format!(
-            "Expected exactly one parameter for {}::{}  of type &mut {}::{}::{}",
+            "Expected last (and at most second) parameter for {}::{} to be &mut {}::{}::{}; \
+             optional first parameter must be of one-time witness type whose name is \
+             the same as the capitalized module name",
```

Comment:
maybe just spell it out for this case, something like `"... e.g. {}::{}::{}", address, module_name, module_name.to_upper()`